plugins/auth/index.js

```diff
@@ -113,15 +113,30 @@ exports.register = (server, options, next) => {
                 accessTokenName: 'access_key',
                 allowCookieToken: false,
                 allowQueryToken: true,
-                validateFunc: (token, cb) => {
-                    if (token !== pluginOptions.temporaryAccessKey) {
-                        return cb(null, false, {});
+                validateFunc: function _validateFunc(token, cb) {
```

Comment:
Arrow notation doesn't bind the request object to `this`

The only thing I need the request object for is to get access to the userFactory though, so if there's a better way to get at that then there's no need for `function` notation.